### PR TITLE
some ideas

### PR DESCRIPTION
**Pull request by zpalmquist**
that didn't make anything work but might have some good stuff in here, just check it out

### CODE REVIEW

File: lib/invoice_repo.rb [hunk 1]
```diff
@@ -29,6 +29,12 @@ def find_all_by_customer_id(id)
    end
  end
 
+ def find_all_successful_transactions_by_merchant_id(id)
```
Comment (lnchambers) [inline lib/invoice_repo.rb:32]: Keep this code on your local, we may need it after we track enumerables

**Review by lnchambers — APPROVED** [review]

File: lib/invoice_repo.rb (hunk 1 above)
Comment (lnchambers) [reply lib/invoice_repo.rb]: image

**Review by lnchambers — CHANGES_REQUESTED** [review]
image

File: lib/invoice_repo.rb (hunk 1 above)
Comment (zpalmquist) [reply lib/invoice_repo.rb:32]: Cool, I'm gonna start tracking stuff out now and we'll just leave that there.

**Pull request closed without merging**